### PR TITLE
Sackmesser report command

### PR DESCRIPTION
**Pull request by danistrebel**
What's changed, or what was fixed?

- Sackmesser report generation

- [x] I have run all the tests locally and they all pass.
- [x] I have followed the relevant style guide for my changes.

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: tools/apigee-sackmesser/cmd/report/report.sh [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+sackmesser list "organizations/$organization/environments/$environment/deployments?sharedFlows=true" > "$sfdeployments"
+
+echo "<div><table id=\"proxy-lint\" data-toggle=\"table\" class=\"table\">" >> "$report_html"
+echo "<thead class=\"thead-dark\"><tr>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"proxy-name\">Proxy</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"env\">Rev. $environment</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"errors\">Apigeelint Errors</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"lint-warn\">Apigeelint Warnings</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"lint-error\">Apigeelint Report</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"policies\">Numer of Policies</th>" >> "$report_html"
```
Comment (OmidTahouri) [inline tools/apigee-sackmesser/cmd/report/report.sh:145]: typo: Numer

**Review by OmidTahouri — APPROVED** [review]
LGTM

**Review by seymen — APPROVED** [review]

**Pull request merged**